tutorials/MeshingExamples/MatIDsBndryIDFromLua.lua

```diff
@@ -10,7 +10,7 @@ for i = 1, (N + 1) do
 end
 
 meshgen1 = mesh.OrthogonalMeshGenerator.Create({ node_sets = { nodes, nodes, nodes } })
-mesh.MeshGenerator.Execute(meshgen1)
+meshgen1:Execute()
 
```

Comment:
We should gather all names that are forbidden to use in the lua input (like `mesh`, `xs`,...) and make them super apparent at the beginning of the tutorials' page or the future user manual's page. can you create an issue and list all forbidden names?

Reply:
issue #454 will deal with this.